dlint/util.py

```diff
@@ -1,21 +1,11 @@
 #!/usr/bin/env python
 
 import abc
-import sys
 
-if sys.version_info >= (3, 4):
-    ABC = abc.ABC
-else:
-    ABC = abc.ABCMeta(str('ABC'), (), {})
+ABC = abc.ABC
 
 
 def lstartswith(l1, l2):
     if len(l2) > len(l1):
         return False
     return l1[:len(l2)] == l2
-
-
-def lendswith(l1, l2):
```

Comment:
Let's leave this in for now. I know it's unused, but it's a potentially useful function to have alongside `lstartswith`.